javascript/javascript3/week1/giphy-searcher/main.js

```diff
@@ -0,0 +1,25 @@
+const key = 'Qp6RAAaBZBQFCXToJtyiMN9akx4BjTDR';
+
+const inputClick = document.getElementById('inputClick');
+const imagelist = document.getElementById('image-list');
+
+const displayImages = () => {
+  const searchgif = document.getElementById('search').value;
+  const numbergif = document.getElementById('number').value;
+  imagelist.innerHTML = '';
+
```

Comment:
Just check for empty user input , before fetching API,

Meaning you should check if serchgif and numbergif is not null or undefined or empty